less/quicksilver/europe2020/drupal.less

```diff
@@ -19,6 +19,11 @@
 @import "_variables.less";
 @import "template.less";
 
+@import "../../../../owl.carousel/dist/assets/owl.carousel.min.css";
```

Comment:
Please use @import '~/owl.carousel...

Reply:
Done

Reply:
After some testing this was the correct path to use: ~[path]

Instead of: ~/[path]

Reply:
Correct!